### PR TITLE
feat(FWS-15): Frontend React Template Setup

### PR DESCRIPTION
**Pull request by AlexeiTimbro**
**JIRA: link to jira ticket**
https://champlainsaintlambert.atlassian.net/browse/FWS-15

**What is the ticket about and why are we making this change?**

This ticket is about making the react template for the fitness package page that will be shown as the Home page of our website when anybody first loads the website. This will help demonstrate the get-all fitness packages method in the environment in which it would be used.

**What are the various changes and what other modules do those changes affect?**
 
-Changed the FitnessService.js to make the whole structure of the page
-Created a style.css file that holds all the CSS for that page because there was no CSS before.

**UI Change:**

Before:
image

After: 
image
image

### CODE REVIEW

File: fitwsarah_frontend/src/models/FitnessService.js [hunk 1]
```diff
@@ -21,16 +23,55 @@ function FitnessServiceList() {
...
+                            </div>
+                        </Col>
+                    ))}
+                </Row>
+            </Container>
+        </section>
+
+        {/* Include other sections like 'appointments-section' and 'packages-section' as needed */}
+
+        <footer className="main-footer">
```
Comment (Eli293) [inline fitwsarah_frontend/src/models/FitnessService.js:60]: Aren't we supposed to have more inside the footer?

File: fitwsarah_frontend/src/models/FitnessService.js [hunk 2]
```diff
@@ -21,16 +23,55 @@ function FitnessServiceList() {
...
+        </header>
+
+        <section className="hero-section">
+            {/* Optionally, if you want to overlay text on the hero image, you can do so here */}
+        </section>
+
+        <section className="services-section">
+            <Container>
+                <h2>Services & Prices</h2>
+                <Row>
```
Comment (Eli293) [inline fitwsarah_frontend/src/models/FitnessService.js:43]: What is this supposed to do?

**Review by Abdur-Siddiqui — APPROVED** [review]
Good work

File: fitwsarah_frontend/src/models/FitnessService.js (hunk 1 above)
Comment (AlexeiTimbro) [reply fitwsarah_frontend/src/models/FitnessService.js:60]: Yes, we are supposed to have the social media platform icon link that will be placed right above the footer. I haven't found the right images yet that would fit the theme the most. They will be added in the future sprints.

File: fitwsarah_frontend/src/models/FitnessService.js (hunk 2 above)
Comment (AlexeiTimbro) [reply fitwsarah_frontend/src/models/FitnessService.js:43]: This is supposed to set the class name of the section to make it easier to call when we use CSS.

**Review by Eli293 — APPROVED** [review]
Looks good, all my questions were answered.

**Pull request merged**